test/unit/validate_all.spec.js

```diff
@@ -43,11 +43,12 @@ function test(file) {
       });
     }
 
-    Object.keys(appParams).forEach(function(param) {
-      it('should validate ":'+param+'"', function() {
-        assert.equal(appParams[param], 'ok');
+
+      Object.keys(appParams).forEach(function(param) {
+        test('should validate ":'+param+'"', () => {
```

Comment:
template string

Reply:
I won't touch inner unit testing code, that will come later in a future refactor phase. It's too much now.